lib/ingram_micro.rb

```diff
@@ -15,4 +16,8 @@ def self.configure
     yield configuration
     configuration.assert_valid
   end
+
+  def self.generate_order_number(prefix='')
```

Comment:
@dl103 is right, we'll need to enforce arg is a Integer. Some doc why it returns a string and why there's an random suffix added to the end would be appreciated 🍰